### PR TITLE
new package: lux 1.2

### PR DESCRIPTION
**Pull request by a-schaefers**
## Description of package
POSIX Shell script to easily control brightness on backlight-controllers.
## Checklist

- [x] Latest upstream version.
- [x] I agree to maintain this package (**required**).
- [x] I agree to notify KISS if I can no longer maintain this package (**required**).

### CODE REVIEW

File: community/lux/build [hunk 1]
```diff
@@ -0,0 +1,7 @@
+#!/bin/sh -e
+
+mkdir -p "$1/usr/bin"
+mkdir -p "$1/etc/udev/rules.d"
```
Comment (dylanaraps) [inline community/lux/build]: Remove these two lines.

File: community/lux/build [hunk 2]
```diff
@@ -0,0 +1,7 @@
...
+
+install -m755 "lux.sh" "$1/usr/bin/lux"
+install -m755 "99-lux.rules" "$1/etc/udev/rules.d/99-lux.rules"
```
Comment (dylanaraps) [inline community/lux/build]:
```sh

install -Dm755 "lux.sh" "$1/usr/bin/lux"
install -Dm755 "99-lux.rules" "$1/etc/udev/rules.d/99-lux.rules"
```

Comment (a-schaefers) [reply community/lux/build]: fixed

File: community/lux/build (hunk 1 above)
Comment (a-schaefers) [reply community/lux/build]: done

**Comment by a-schaefers** [comment]
done

**Comment by a-schaefers** [comment]
rm depends 18619ef

**Comment by dylanaraps** [comment]
The depends file still exists in this PR. :+1:

**Comment by a-schaefers** [comment]
doh

**Comment by a-schaefers** [comment]
merge it

**Pull request merged**